pages/home.md

```diff
@@ -58,8 +58,8 @@ We work with you to assess your current system and determine the most cost-effic
     </button>
     <div id="assessment-content" class="usa-accordion-content">
       <ul>
-        <li>We’ll ensure your Standards implementation is in compliance with the official web policy guidance from OMB and your agency.</li>
-        <li>We’ll determine any gaps in skill sets, equipment, design, and code quality needed for successful implementation of the Standards.</li>
+        <li>We’ll ensure your Design System implementation is in compliance with the official web policy guidance from OMB and your agency.</li>
```

Comment:
`is in compliance with the` s/b `conforms to`

Reply:
If this is about following the letter of the law, then compliance is still OK.

Reply:
I'm sure either are OK and get the job done, but as I understand it people, agencies, or other intentional agents comply with laws by writing code, producing products, or, in essence producing outputs that conform with the letter of the law. I may be wrong about this, but that's how I've distinguished them in my head!

Reply:
I can approve this PR, tho :)